src/internal.c

```diff
@@ -33986,8 +33986,18 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         #endif
             /* Resumption master secret. */
             XMEMCPY(it->msecret, ssl->session->masterSecret, SECRET_LEN);
+#ifdef WOLFSSL_TICKET_NONCE_MALLOC
+            if (ssl->session->ticketNonce.data !=
+                    ssl->session->ticketNonce.dataStatic ||
+                ssl->session->ticketNonce.len >
+                    sizeof(ssl->session->ticketNonce.dataStatic)) {
```

Comment:
no `ticketNonce.len` is where the real length of the nonce is stored and is copied together when copying the `TicketNonce` struct. We as a server do not support longer tickets than the size of the static buffer. This should never happen btw because `ticketNonce.Len` is always 1 in wolfSSL server